examples/coherency_example_2.py

```diff
@@ -0,0 +1,172 @@
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+from solcore import siUnits, material, si
+from solcore.interpolate import interp1d
+from solcore.solar_cell import SolarCell
+from solcore.structure import Junction, Layer
+from solcore.solar_cell_solver import solar_cell_solver, default_options, prepare_solar_cell
+from solcore.light_source import LightSource
+from solcore.state import State
+
+def this_dir_file(f):  # this is just Python stuff, so it can find external files containing optical constant data which are in the same folder as this file.
+    return os.path.join(os.path.split(__file__)[0], f)
+
+incidence_angle = 45 # should be in degrees
+wl = np.linspace(290, 1900, 400) * 1e-9
+concX=566 # the light concentration
```

Comment:
Out of curiosity, why this very specific number?

Reply:
Because it is based on one of the cell designs in [this paper](https://www.sciencedirect.com/science/article/pii/S092702481930011X), where we are considering a concentration of 800 incident on an Si cell and then reflected onto the triple-junction at a 45 degree angle, so due to the geometry the effective concentration is 800/sqrt(2) = 565.7.